menas/src/main/scala/za/co/absa/enceladus/menas/repositories/VersionedMongoRepository.scala

```diff
@@ -55,42 +63,38 @@ abstract class VersionedMongoRepository[C <: VersionedModel](mongoDb: MongoDatab
   }
 
   def getLatestVersionValue(name: String): Future[Option[Int]] = {
-    val pipeline = Seq(
-      filter(getNameFilter(name)),
+    val pipeline = Seq(filter(getNameFilter(name)),
       filter(getNotDisabledFilter),
-      Aggregates.group("$name", Accumulators.max("latestVersion", "$version"))
-    )
+      Aggregates.group("$name", Accumulators.max("latestVersion", "$version")))
     collection.aggregate[VersionedSummary](pipeline).headOption().map(_.map(_.latestVersion))
   }
 
   def getAllVersions(name: String, inclDisabled: Boolean = false): Future[Seq[C]] = {
-    val filter = if(inclDisabled) getNameFilter(name) else getNameFilterEnabled(name)
+    val filter = if (inclDisabled) getNameFilter(name) else getNameFilterEnabled(name)
     collection.find(filter).toFuture()
   }
 
   def create(item: C, username: String): Future[Completed] = {
     super.create(item
       .setCreatedInfo(username)
       .setUpdatedInfo(username)
-      .asInstanceOf[C]
-    )
+      .asInstanceOf[C])
   }
 
   def update(username: String, updated: C): Future[C] = {
     for {
       latestVersion <- getLatestVersionValue(updated.name)
-      newVersion <- if(latestVersion.isEmpty) throw new NotFoundException()
-           else if(latestVersion.get != updated.version) throw new EntityAlreadyExistsException(s"Entity ${updated.name} (version. ${updated.version}) already exists.") 
-           else Future.successful(latestVersion.get + 1)
+      newVersion <- if (latestVersion.isEmpty) throw new NotFoundException()
+      else if (latestVersion.get != updated.version) throw new EntityAlreadyExistsException(s"Entity ${updated.name} (version. ${updated.version}) already exists.")
```

Comment:
Please add some braces and new lines. + the exceptions are case classes so `new` is not needed